app/components/windows/AddSourceInfo.vue

```diff
@@ -14,7 +14,6 @@
     <div class="source-info__media">
       <slot name="media"></slot>
     </div>
-    <!--<button @click="$emit('clickAdd')" class="button button--action">Add Source</button>-->
```

Comment:
If we're removing this can we please delete it rather than commenting it out?

Reply:
This is a deletion.

Reply:
I apparently hadn't had enough of my coffee this morning when I posted this.  Oops.  😴